docs/demography.md

```diff
@@ -1795,16 +1795,16 @@ demography.set_migration_rate(source="B", dest="A", rate=0.02)
 
 debug = demography.debug()
 T = np.linspace(0, 500, 51)
-RAA, _ = debug.coalescence_rate_trajectory(T, [2, 0])
-RBB, _ = debug.coalescence_rate_trajectory(T, [0, 2])
-RAB, _ = debug.coalescence_rate_trajectory(T, [1, 1])
+RAA, _ = debug.coalescence_rate_trajectory(T, {"A": 2})
+RBB, _ = debug.coalescence_rate_trajectory(T, {"B": 2})
+RAB, _ = debug.coalescence_rate_trajectory(T, {"A": 1, "B": 1})
```

Comment:
need ploidy=1

Reply:
It's pretty ugly and confusing then though, if we have specify the SampleSet thing every time.